### PR TITLE
zh-translation: content/zh/news/releases/1.5.x/_index.md   .....

### PR DESCRIPTION
**Pull request by kylesliu**
Ref: https://github.com/servicemesher/istio-official-translation/issues/1589
Ref: https://github.com/servicemesher/istio-official-translation/issues/1590
Ref: https://github.com/servicemesher/istio-official-translation/issues/1591
Ref: https://github.com/servicemesher/istio-official-translation/issues/1592

### CODE REVIEW

File: content/zh/news/releases/1.5.x/_index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: 1.5.x Releases
```
Comment (gorda) [inline content/zh/news/releases/1.5.x/_index.md]: > 1.5.x Releases

建议翻译。

File: content/zh/news/releases/1.5.x/_index.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
-description: Announcements for the 1.5 release and its associated patch releases.
+description: 1.5 版本及其相关修补程序版本的公告。
```
Comment (gorda) [inline content/zh/news/releases/1.5.x/_index.md]: > 1.5 版本及其相关修补程序版本的公告。

建议：1.5 版本及其相关修补程序发布的公告。

File: content/zh/news/releases/1.5.x/announcing-1.5/_index.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 ---
-title: Announcing Istio 1.5
+title: 宣布 Istio 1.5 发布
 linktitle: 1.5
-subtitle: Major Update
-description: Istio 1.5 release announcement.
+subtitle: 重大更新
+description: Istio 1.5发布公告。
```
Comment (gorda) [inline content/zh/news/releases/1.5.x/announcing-1.5/_index.md]: > Istio 1.5发布公告。

建议（只是加个空格）：Istio 1.5 发布公告。

**Comment by kylesliu** [comment]
暂时不用翻译。

**Pull request closed without merging**